peachjam/views/legislation.py

```diff
@@ -17,3 +23,46 @@ class LegislationListView(FilteredDocumentListView):
 class LegislationDetailView(BaseDocumentDetailView):
     model = Legislation
     template_name = "peachjam/legislation_detail.html"
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context["notices"] = self.get_notices()
```

Comment:
This can go into the superclass, since that's where the default implementation of `get_notices` is.